helsingborg-stad/app-mitt-helsingborg
# remove hardcoded StepConfirmation component

## Explain the changes you’ve made
Remove the stop-gap StepConfirmation component, which added some information cards to the last page of every form. 

## Explain why these changes are made

Hardcoded solutions are bad, and we should be better.

## Explain your solution

Remove the file, and the usage of it in StepDescription. 
Then, in the formbuilder I added dynamical cards with the same content to the last pages of EKB löpande and EKB Löpande AMF, so for these forms the last page should still look the same. 

The only thing that doesn't quite work is the navigation button going to the right place, since the dynamical cards at the moment do not support navigating to nested screens. So that's something to add. Otherwise the cards should look the same. 

## How to test the changes?
Before checking out the branch, on develop, you can go into one of the EKB Löpande forms, and go to the last page. Then there should be two copies of cards, where the first is from the hardcoded and the second is dynamic. Check if they are the same. 

Then check out this branch, and check that there's only one copy of the cards. 

## Was this feature tested in the following environments?
- [] Storybook on a iOS device/simulator.
- [] Building the Application on a Android device/simulator.
